File: _inc/_DEPRECATED_django/erp_prestech_backend/app/Models/_helpers/worker.py
from django.db import models
from .person import Person
from .fields import VOID, VALID_MYSQL_MIN_DATE
class Worker(Person):
  class Meta:
    abstract = True
  dob = models.DateField(validators=[VALID_MYSQL_MIN_DATE], **VOID)
  skill = models.TextField(max_length=65535, **VOID)

File: _inc/laravel/app/Models/Abstracts/Entities/Worker.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;

abstract class Worker extends Person
{
  use HasFactory;
  protected $casts = [
    ...parent::CASTS,
    'dob' => 'date',
  ];
  protected $fillable = [
    ...parent::FILLABLE,
    'dob',
    'skill',
  ];
}
